installer/packer/scripts/systemd/kov/kov.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Kubernets on Vsphere
+Documentation=https://github.com/supervised-io/kov
+After=kov_startup.service
```

Comment:
I make kov.service after admiral.service and admiral.service is after docker.service...